ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-main.yaml

```diff
@@ -273,6 +273,45 @@ tests:
   container:
     from: operator-sdk-mto
   skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Za-z]+\.md$)|((^|/)\.github/)|((^|/)\.tekton/)
+- always_run: false
+  as: gcp-ovn-mto-heterogeneous-perfscale
+  cluster: build10
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-initial
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: n2-standard-4
+      ADDITIONAL_WORKERS: "12"
+      COMPUTE_NODE_REPLICAS: "12"
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      MTO_OPERATOR_INSTALL_METHOD: catalog
+      OCP_ARCH: arm64
+    test:
+    - ref: ipi-install-heterogeneous
+    - ref: multiarch-tuning-operator-deploy
+    - ref: multiarch-tuning-operator-deploy-pod-placement
+    - ref: openshift-qe-node-density
```

Comment:
I'm working on exposing the same information in https://github.com/openshift/release/pull/55603/files#r1744367854 as gauges/histogram Prometheus metrics and checking if we can onboard them in the cluster-monitoring operator. See https://issues.redhat.com/browse/MULTIARCH-4294


Based on our previous conversation, there should be a way to obtain the `time to schedule` (from the creation time to the pod having a value for .spec.nodeName). 

Is there also a way to catch custom metrics (coming from the Prometheus instance of the cluster-monitoring operator) in your grafana dashboards?

Reply:
I need some time to study how to gather 'customized' metrics and show them on grafana dashboard.